File: SLAManager/SLAManager/EntityFramework/Models/SlaMetricStatus.cs
﻿using System;
using System.Collections.Generic;

namespace SLAManagerdata.Models;

public partial class SlaMetricStatus
{
    public int Id { get; set; }

    public int? IdSla { get; set; }

    public int? IdStatus { get; set; }

    public string? Action { get; set; }

    public string? Code { get; set; }

    public string? Controller { get; set; }

    public string? Endpoint { get; set; }

    public string? Instance { get; set; }

    public string? Job { get; set; }

    public string? Method { get; set; }

    public DateTime? Datetime { get; set; }

    public double? MisuredValue { get; set; }

    public double? DesiredValue { get; set; }
}

File: SlaManagerService/app/Model/SlaMetricStatus.py
from datetime import datetime


class SlaMetricStatus:
    def __init__(self, IdSla=None, IdStatus=None,Action=None,Code=None,Controller=None,Endpoint=None,Instance=None,Job=None,Method=None,DesiredValue=None,MisuredValue=None):
        self.IdSla = IdSla
        self.IdStatus = IdStatus
        self.DesiredValue = DesiredValue
        self.MisuredValue = MisuredValue 
        self.Datetime = datetime.utcnow()
        self.Action = Action
        self.Code = Code
        self.Controller = Controller
        self.Endpoint = Endpoint
        self.Instance = Instance
        self.Job = Job
        self.Method = Method
